### ISSUE TITLE
Connecting through proxy to insecure sense 

### ISSUE THREAD

**Issue by axelssonHakan** [qlik-oss/corectl#450]
When using `corectl` to connect to a Sense Enterprise with self-signed cert the ws fails with:
`Details: x509: certificate signed by unknown authority`

Maybe we should add an option to set `--insecure` to allow connecting using self-signed certs?

https://github.com/qlik-oss/corectl/blob/master/internal/state.go#L53
```
var dialer = enigma.Dialer{}

tlsConf := tls.Config{
  InsecureSkipVerify: true,
}

dialer.TLSClientConfig = &tlsConf
```

**Comment by axelssonHakan** [comment]
This seems to be done already if certs are used
https://github.com/qlik-oss/corectl/blob/master/internal/config.go#L173

**Comment by axelssonHakan** [comment]
Implemented in #463
